I/Ipopt/build_tarballs.jl

```diff
@@ -56,7 +69,7 @@ if [[ "${target}" == *-mingw* ]]; then
 fi
 """
 
-platforms = expand_gfortran_versions(expand_cxxstring_abis(supported_platforms()))
+platforms = expand_gfortran_versions(expand_cxxstring_abis(filter!(p -> !isa(p, Windows), supported_platforms())))
```

Comment:
Why?  It's working now.

Reply:
Sorry, it's an oversight. Any idea about freebsd?

Reply:
> Any idea about freebsd?

If upstream doesn't properly update the `configure` script, no.  It's just a pain to work without a clue about how to regenerate it because they have their own setup to do this.

Reply:
Seems it's been updated in v.3.13.1. Building now.